ERCS/erc-7602.md

```diff
@@ -0,0 +1,71 @@
+---
+eip: 7602
+title: Ethereum Mixing Transaction Supervision
+description: Monitoring and regulating mixing transactions on the Ethereum blockchain.
```

Comment:
```suggestion
title: Mixing Transaction Supervision
description: Monitoring and regulating mixing transactions.
```

All ERCs are related to Ethereum, so mentioning it in the title and description is somewhat redundant.